src/prismic/Fragment.cs

```diff
@@ -484,7 +484,15 @@ public class Embed: Fragment {
 
 		}
 
-		public class Slice
+		public interface ISlice : Fragment
+		{
+			string SliceType { get; }
+			string SliceLabel { get; }
+
+			string AsHtml(DocumentLinkResolver resolver);
+		}
+
+		public class Slice : ISlice
```

Comment:
Sure. Would you like the `ISlice` interface to be renamed to `Slice` in keeping with the current coding style?